src/components/cell.tsx

```diff
@@ -1,12 +1,14 @@
 import React, { useEffect } from 'react';
 import { areEqual } from 'react-window';
 import tw, { TwStyle } from 'twin.macro';
-import anchorme from 'anchorme';
+import Linkify from 'linkify-it';
 import { cellTypeMap } from '../store';
 import { DashIcon, DiffModifiedIcon, PlusIcon } from '@primer/octicons-react';
 import DOMPurify from 'dompurify';
 import { EditableCell } from './editable-cell';
 
+const linkify = Linkify().add('ftp:', null).add('mailto:', null);
```

Comment:
I feel like we should avoid linkifying these types of URLs. Open to feedback!

Reply:
agreed, good call!